src/catalog/catalog-menu.md

```diff
@@ -4,44 +4,48 @@ redirect_from:
   - /catalog.html
 ---
 
-The Catalog Menu provides easy access to product creation, category and inventory management tools{% if "Default.EE Only,Default.CE Only" contains site.edition %}.{% endif %} {% if "Default.B2B Only" contains site.edition %}, as well as shared catalogs for custom pricing.{% endif %}
+The Catalog Menu provides easy access to product creation, category, and inventory management tools{% if "Default.EE Only,Default.CE Only" contains site.edition %}.{% endif %} {% if "Default.B2B Only" contains site.edition %}, as well as shared catalogs for custom pricing.{% endif %}
 
 <!--{% if "Default.CE Only" contains site.edition %}-->
 ![]({{ site.baseurl }}{% link images/images/admin-menu-catalog.png %}){: .zoom}
-*Catalog Menu*
+_Catalog Menu_
 <!--{% endif %}-->
 <!--{% if "Default.EE Only" contains site.edition %}-->
 ![]({{ site.baseurl }}{% link images/images-ee/admin-menu-catalog-ee.png %}){: .zoom}
-*Catalog Menu*
+_Catalog Menu_
 <!--{% endif %}-->
 <!--{% if "Default.B2B Only" contains site.edition %}-->
 ![]({{ site.baseurl }}{% link images/images-b2b/admin-menu-catalog.png %}){: .zoom}
-*Catalog Menu*
+_Catalog Menu_
 <!--{% endif %}-->
 
-## To display the Catalog menu
+## Display the Catalog menu
 
-On the Admin sidebar, tap <span class="btn"> Catalog </span>.
+On the Admin sidebar, click **Catalog**.
```

Comment:
@hickskenh Leslie told me to leave button classes. @leslietilling ??

Reply:
Yes, we are leaving buttons as the spans.